package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@rsksmart/rlogin",
-  "version": "0.1.1",
+  "version": "0.2.0",
```

Comment:
```suggestion
  "version": "1.0.0",
```

Reply:
I have updated the version in the commit below. 22d56bc.